ILRepack.Tests/ILRepack.Tests.csproj

```diff
@@ -55,6 +58,15 @@
     <None Include="packages.config">
       <SubType>Designer</SubType>
     </None>
+    <None Include="sharpdevelop\ICSharpCode.SharpDevelop.BuildWorker120.exe.config" />
```

Comment:
Yes, I just added this pull request to showcase and have a discussion about using parallel linq and performance test. Don't want to merge this. 

I also don't want to add all those binaries to the repository. The proper way would be to add a gradle script that gets these files from sharpdevelop (or any other repository with a lot of dlls? :)?. And I plan on doing that eventually. 

With the tests I want to spot bottlenecks and evaluate the performance boost of switching to parallel linq.